alf/algorithms/rl_algorithm.py

```diff
@@ -598,19 +605,62 @@ def _async_unroll(self, unroll_length: int):
 
         return experience
 
+    def should_post_process_episode(self, rollout_info, step_type: StepType):
+        """A function that determines whether the ``post_process_episode`` function should
+        be applied to the current list of experiences.
+        """
```

Comment:
This is an interface mainly used for subclasses? Maybe mention this.

Reply:
Good point. Added comments. Also for post_process_episode